Documentation/user-manual.txt

```diff
@@ -3714,7 +3735,11 @@ image::images/Planner_pSCR.jpg["FIGURE: Planning a pSCR dive: setup",align="cent
 [NOTE]
 Open circuit diving is often part of a pSCR dive because, at shallow depths, the pO~2~ in the loop is often
 too low for breathing and the shallow parts of a pSCR dive are often done on OC. The change
-from rebreather mode to OC or _vice versa_ is indicated using bailout events. The image on the
+from rebreather mode to OC or _vice versa_ is indicated using bailout events. There are two ways to plan
+bailout in rebreather dives: 1) Check the _Bailout_ checkbox in the _Dive mode_ section of the planner. In this
+case, bailout requirements will be calculated starting at the *end of the deepest part* of the dive. 2) If
```

Comment:
no. At the end of the manually entered part of the dive.

Reply:
Done